templates/citizen_action_manager/contact_participants.html.twig

```diff
@@ -39,7 +39,7 @@
 
                 {{ form_start(form) }}
 
-                   {{ form_widget(form.contacts, { value: contacts|json_encode }) }}
+                    <input type="hidden" name="contacts" value="{{ contacts|json_encode }}" />
```

Comment:
Why removing the field from the form type ? Is it just a way of giving some data to some javascript code ?
In that we should completely remove this field no ? 
If we keep the form. Is the name need to be the same as the form ? (in that case we should `vars` from the `form`)

Reply:
I explained in the PR comment why I do this. 
We need `contacts` in the controller action. The datas posted to this action are different depending on page accessing this action. Actually I can't find more propre/simple solution than this one.